app/mailers/notify_mailer.rb

```diff
@@ -48,6 +48,19 @@ def revive_account_email(email)
     mail(to: email)
   end
 
+  def role_changed_email(emails, user)
+    set_template(:role_changed_email)
+    set_email_reply_to(:onboarding_reply_to_address)
+
+    set_personalisation(
+      user_name: user.name,
+      role_name: user.role.humanize,
+      onboarding_email: Rails.configuration.x.admin.onboarding_email
```

Comment:
I am wondering if this should be the intercept email address rather than this one?